### PR TITLE
Create php.yml

### PR DESCRIPTION
**Pull request by jayay**

### CODE REVIEW

**Review by legionth — APPROVED** [review]
Noice!

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Setup PHP with pre-release PECL extension
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: '7.4'
```
Comment (legionth) [inline .github/workflows/php.yml:20]: Should there be more supported versions here ? I don't know the exact specs for this lib, but I think this Lib supports a bunch of PHP Versions.

Comment (jayay) [reply .github/workflows/php.yml]: I've added PHP 7.0, 7.1, 7.2, 7.3 and 7.4. These match the version constraints of PHPUnit v6.x and php-cs-fixer v2.16.

Comment (jayay) [reply .github/workflows/php.yml]: Should I add a cron check as well?

Comment (jayay) [reply .github/workflows/php.yml]: I'mma do it. It'll make a it a little bit safer

**Review by legionth — APPROVED** [review]
:shipit:

**Review by dkaisers — APPROVED** [review]
🚢 🇮🇹

**Pull request merged**